src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+import java.io.IOException;
+
+/**
+ * This Duke program is a chatbot that takes handles a list for the user.
+ */
+
+public class Duke {
+
+    private UI ui;
+    private Storage storage;
+    private Parser parser;
+    private TaskList tasks;
+    public Duke() throws IOException {
+        ui = new UI();
+        storage = new Storage();
+        parser = new Parser();
+
+        try {
+            tasks = new TaskList(storage.loadTasks());
+        } catch (NullPointerException e) {
+            tasks = new TaskList();
+        }
+    }
+
+    public void run() throws IOException {
+
+        ui.startMsg();
+
+        boolean finish = false;
```

Comment:
Considering code standard, isn't isFinished() a better way of naming this boolean variable?